### ISSUE TITLE
P2b - Code HTML pages

### ISSUE THREAD

**Issue by louis-0** [PaulShunOSAWA/web-develop-t1g1-aqt#10]
Code the three pages in HTML and enter in some essential text content that you write yourself. Name the main/first page of your web site: index.html. Ensure that you use comments to help all the coders to understand the code.

Find some pictures and any other media that you consider appropriate and embed them into the pages using the correct HTML tags. Ensure that your code includes comments to help you work with the code and quickly relate it to the wireframes. It could be helpful to include comments to indicate who contributed chunks of the code so that you can ensure you can work together as a team.

Criteria 2: The range of different layout, formatting and media HTML tags used and their
appropriate use (eg tables used as tables not for layout)
• Significant amount of missing layout, formatting and media HTML tags (0 pts)
• Good range of different layout, formatting and media HTML tags used and their
appropriate use, including layout structures such as lists, a variety of media and a
range of formatting tags beyond simple h1 and p tags only. They used the
tags taught in the course but not more. (0.5 pts)
• Extensive and innovative use of a wide range of different layout, formatting and
media HTML tags used and their appropriate use, with good use of layout
structures such as lists, extended and appropriate range of media and use of
formatting tags that demonstrate a good understanding of the value in formatting
content richly. They have gone beyond what was taught in the course. (1 pt)

Bundled Submission Criteria:
7. Rich and appropriate selection of HTML elements
• Few different tags used, with missed opportunities to use more specific tags. May
have misuse of tags (e.g. tables may be used for layout, or deprecated formatting
elements like <center>). (0 pts)
• Good range of different elements used appropriately. (1 pts)
• Extensive and appropriate use of a wide range of different elements. This should
include media elements as well as text structuring elements. This use makes styling
the site cleaner and easier. (2 pts)

**Comment by Apirich** [comment]
Shun: So I just took a look at the index.html. I adjusted some of the indent to make it neater, and I considered some of these issues:

- The structure looks good but if the navigation bar has 5 tabs and 4 of it link to the same page then it is a little confusing. And if you want to link it to the same page, then may be should add #id to the href to point it to the part it means to be.

- The Q&A we may use collapsible button to make it look neater, and so our code can have variety of html tags.

- We also should add more images, black and white drawing images will make a theme park site look boring.

**Comment by Apirich** [comment]
Btw, I just saw To do list mentioned that Shun and Louis discussed. Why only Shun and Louis discuss about What to do list? I saw nothing about discussing in our group chats, Slackbot, Trello, and even Github, nothing at all about discussing or schedule about discussing at all?

**Comment by PaulShunOSAWA** [comment]
The structure looks good but if the navigation bar has 5 tabs and 4 of it link to the same page then it is a little confusing. And if you want to link it to the same page, then may be should add #id to the href to point it to the part it means to be.
→Thanks a lot! I corrected it.

The Q&A we may use collapsible button to make it look neater, and so our code can have variety of html tags.
→I added this to TO DO lists. Thanks.

We also should add more images, black and white drawing images will make a theme park site look boring.
→As I mentioned a month ago, this is just a sample image in order to see the wireframe Louis gave us.
